### DISCUSSION TITLE
net40 Dlls

### DISCUSSION THREAD

**Discussion by zrkolle** [dlemstra/Magick.NET#1057] [Help]
Hello, what happened with net40 DLLs? There is no such folder under lib folder in the latest release?

Thanks
Zack

**Comment by dlemstra** [comment]
Those have been removed and you will either need to use the `net20` or the `netstandard20` version of the library. See release notes of version `8.0.0`:  https://github.com/dlemstra/Magick.NET/releases/tag/8.0.0
